ArduPlane/systemid.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include "quadplane.h"
+
+#ifndef AP_PLANE_SYSTEMID_ENABLED
+#define AP_PLANE_SYSTEMID_ENABLED HAL_PROGRAM_SIZE_LIMIT_KB > 1024 && HAL_QUADPLANE_ENABLED
+#endif
+
+#if AP_PLANE_SYSTEMID_ENABLED
+
+#include <AP_Math/chirp.h>
+
+class AP_SystemID {
+
+public:
+    AP_SystemID(void);
+    void start(void);
+    void stop(void);
+    void update();
+
+    static const struct AP_Param::GroupInfo var_info[];
+
+    const Vector3f &get_attitude_offset_deg(void) const {
+        return attitude_offset_deg;
+    }
+    float get_throttle_offset(void) const {
+        return throttle_offset;
+    }
```

Comment:
Because this is not called right next the `update` function like `get_attitude_offset_deg` is I would like it to be clear that it will return 0 if not running. 

```suggestion
    float get_throttle_offset(void) const {
        return running ? throttle_offset : 0.0;
    }
```